resources/resources.rst

```diff
@@ -0,0 +1,84 @@
+.. _resources:
+
+Resources
+================
+The resources provided in the ``resources`` folder are available for use under CC-BY license |CC-BY|_.
+The original authors are named either in the documents or in the list down below.
+
+Any files submitted to the documentation will be "licensed" under ``ros2_control`` as organization and stating your name, e.g., ``CC-BY My Name (ros2_control)``.
+
+Presentations
```

Comment:
shouldnt' we add yours and Andy's presentations from ROS World?

Reply:
Yeah, we could probably also do this.